cinnabar/measurements.py

```diff
@@ -53,19 +54,55 @@ class Measurement(DefaultModel):
 
     @classmethod
     def from_experiment(cls,
-                        pIC50: float,
+                        Ki: unit.Quantity,
                         label: str,
+                        uncertainty: unit.Quantity = 0 * unit.nanomolar,
                         source: str = '',
+                        temperature: unit.Quantity = 298.15 * unit.kelvin,
                         ):
         """Create Measurement from experimental data
 
         Parameters
         ----------
+        Ki: unit.Quantity
+            experimental Ki value
+            ex.: 500 * unit.nanomolar OR 0.5 * unit.micromolar
+        label: str, optional
+            label for this data point.
+        uncertainty: unit.Quantity
+            uncertainty of the experimental value
+            default is zero if no uncertainty is provided (0 * unit.nanomolar)
+        source: str, optional
+            source of experimental measurement
+        temperature: unit.Quantity
+            temperature in K at which the experimental measurement was carried out.
+            By default: 298 K (298.15 * unit.kelvin)
         """
+        R = 1.9872042586408 * 0.001 * unit.kilocalorie_per_mole / unit.kelvin
+        # Convert Ki and uncertainty to molar, then strip unit off for the math
+        Ki = Ki.to(unit.molar).m
+        uncertainty = uncertainty.to(unit.molar).m
+        if Ki > 0:
+            # dG = RT ln Ki
+            DG = R.m * temperature.m * math.log(Ki) * unit.kilocalorie_per_mole
```

Comment:
```suggestion
        if Ki > 0 * unit.molar:
            DG = (unit.molar_gas_constant * temperature.to(unit.kelvin)
                  * math.log( Ki / unit.molar)).to(unit.kilocalorie_per_mole)
```
I'm just checking against what the PLB returns and I noticed we've got a 2 d.p. discrepancy. I _think_ it's because we're using slightly different values for `R`. We should pin this to the openff.units value just so that we can propagate changes where needed.

Reply:
Yes, that looks good!